R/fit_baseline_one_location.R

```diff
@@ -0,0 +1,228 @@
+#' Fit a baseline model  for one location
+#'
+#' Get quantile function
+#'
+#' @param  predictions baseline predictions
+#' @param  taus probability levels
+#'
+get_quantiles_df <- function(predictions, taus) {
+  n <- length(taus)
+  purrr::map_dfr(1:4,
+                 function(h) {
+                   data.frame(
+                     h = rep(h, n),
+                     quantile = taus,
+                     value = ceiling(quantile(predictions[, h], probs = taus))
```

Comment:
Suggest rounding here
```suggestion
                     value = round(quantile(predictions[, h], probs = taus))
```